INSTALL.adoc

```diff
@@ -198,3 +198,71 @@ and sanity checks that could help you pinpoint the problem.
   requirements for mathematical accuracy, numerical precision, and proper
   handling of mathematical corner cases and error conditions may need to
   consider running their code on a platform with better C99 support.
+
+== (experimental) Building a cross compiler
+
+A cross compiler is a compiler that runs on some machine, named the
+_host_, but generate code for different machine, named the _target_. To
+build a cross compiler you first need to have a non-cross compiler of
```

Comment:
```suggestion
build a cross compiler you first need to have a native compiler of
```

Reply:
I had looked for a standard way to say “non-cross” but didn’t encounter something I find non-ambiguous. As “native“ already refers to the machine-code compiler, I’m not sure that’s a better phrasing.